src/main/java/seedu/duke/commands/DeleteCommand.java

```diff
@@ -49,18 +49,18 @@ public DeleteCommand(int taskIndex, String taskModule) {
     public CommandResult execute(ModuleList moduleList, Configuration configuration) throws ModHappyException {
         if (taskIndex < 0) {
             deleteModule(moduleList);
+            return new CommandResult(result);
+        }
```

Comment:
I suggest change the magic logic here, at least add comments explaining what taskIndex < 0 mean

Reply:
Also the EditCommand remains unchanged. https://github.com/ngys117/tp/blob/282df85449c6c823bfcc5de7da3ce1e1108924d7/src/main/java/seedu/duke/commands/EditCommand.java#L105-L107
<img width="928" alt="Screen Shot 2022-04-03 at 10 48 11 PM" src="https://user-images.githubusercontent.com/78465180/161433663-94fa1a80-adfb-4664-886f-bdb6a43839eb.png">

Reply:
Ah, missed the edit bug. Thanks for the catch.